### PR TITLE
Add Processor Support

### LINKED ISSUE #10393: Add support of Processors in all the Observability Integrations
[Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) are used to provide the ability to the user to enrich their data. 
Currently, there is inconsistency in the Integrations wrt availability of Processors.

- [ ] Identify the Packages/datastreams missing the Processors
- [ ] Do bulk addition of Processors to the Integrations
- [ ] Test the changes

Expectation: Processors yaml box like below should be present in all Integrations
Screenshot 2024-07-05 at 12 12 16 PM

### PR DESCRIPTION
**Pull request by Linu-Elias**
## Proposed commit message
Adding Processor Support for the following Packages

- Oracle WebLogic
- PostgreSQL
- Prometheus Input
- RabbitMQ
- Redis
- Redis Enterprise
- Spring Boot
- Traefik
- Zookeeper

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Relates #10393

### CODE REVIEW

File: packages/rabbitmq/changelog.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
+
```
Comment (harnish-elastic) [inline packages/rabbitmq/changelog.yml:1]: Minor nit: remove extra line!

File: packages/rabbitmq/changelog.yml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+  changes:
+    - description: Add processor support for connection, exchange, node and queue data streams.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10411
 - version: "1.15.0"
   changes:
     - description: ECS version updated to 8.11.0. Update the kibana constraint to ^8.13.0. Modified the field definitions to remove ECS fields made redundant by the ecs@mappings component template.
       type: enhancement
       link: https://github.com/elastic/integrations/pull/10171
+
```
Comment (harnish-elastic) [inline packages/rabbitmq/changelog.yml:12]: Minor nit: remove extra line!

File: packages/spring_boot/changelog.yml [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+  changes:
+    - description: Add processor support for gc, memory and threading data streams.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10411
 - version: "1.6.0"
   changes:
     - description: ECS version updated to 8.11.0. Update the kibana constraint to ^8.13.0. Modified the field definitions to remove ECS fields made redundant by the ecs@mappings component template.
       type: enhancement
       link: https://github.com/elastic/integrations/pull/10171
+
```
Comment (harnish-elastic) [inline packages/spring_boot/changelog.yml:12]: Minor nit: remove extra line!

**Review by harnish-elastic — APPROVED** [review]
LGTM! Minor comments given.

File: packages/spring_boot/changelog.yml (hunk 3 above)
Comment (Linu-Elias) [reply packages/spring_boot/changelog.yml]: Done

File: packages/rabbitmq/changelog.yml (hunk 2 above)
Comment (Linu-Elias) [reply packages/rabbitmq/changelog.yml]: Done

File: packages/rabbitmq/changelog.yml (hunk 1 above)
Comment (Linu-Elias) [reply packages/rabbitmq/changelog.yml]: Done

**Review by devamanv — APPROVED** [review]

**Pull request merged**